### ISSUE TITLE
Quick question

### ISSUE THREAD

**Issue by axiand** [stuyy/chat-platform-react#5]
Would you be ok with me trying to make a bot for this project? I'll do all the development on a locally hosted server, but I just wanna make sure there's no problem with me releasing it into one of your test environments.

Also, on the topic, will there ever be an "official" bot api for the app? It's a long-term goal, but doesn't hurt to ask.

**Comment by stuyy** [comment]
I would like to setup a Bot API at some point. The concept is cool and shouldn't be hard. Basically you can have the bot connect to the gateway and then it can start receiving events for the "servers" (which we will call Spaces for now).

**Comment by axiand** [comment]
Indeed. Also, adding servers sounds super cool and I'm excited to see how everything is implemented.
